README.md

```diff
@@ -73,6 +73,8 @@ var easyMDE = new EasyMDE();
 </script>
 ```
 
+If you need to remove installed listeners (when editor not needed anymore), call `easyMDE.cleanup()`
+
```

Comment:
The placement of the documentation is a bit odd, in between the parts describing how to load the editor. Maybe move this to https://github.com/Ionaru/easy-markdown-editor#removing-easymde-from-text-area ?

Reply:
@Ionaru agree, thx, done